process/block/metablock.go

```diff
@@ -2203,7 +2203,7 @@ func (mp *metaProcessor) applyBodyToHeader(metaHdr data.MetaHeaderHandler, bodyH
 		return nil, err
 	}
 
-	totalTxCount, miniBlockHeaderHandlers, err := mp.createMiniBlockHeaderHandlers(body)
+	totalTxCount, miniBlockHeaderHandlers, err := mp.createMiniBlockHeaderHandlers(body, make(map[string]*processedMb.ProcessedMiniBlockInfo))
```

Comment:
It is a common method, for shardProcessor and metaProcessor, located in baseProcess. In metachain we don't use processed/pending mini blocks. There are also other old common methods which are called from metaProcessor with some nil or make(...) parameters, when these parameters are not used further, just to avoid duplicated code.

Reply:
if not used in one of the implementations, can we not pass nil instead where not used (for mp)?